src/app/static/src/app/components/load/LoadErrorModal.vue

```diff
@@ -2,15 +2,48 @@
     <div id="load-error-modal">
         <modal :open="hasError">
             <h4 v-translate="'loadError'"></h4>
-            <p>{{ loadError }}</p>
+            <p v-if="showInvalidSteps">
+              <span v-translate="'loadErrorSteps'" />
+              <ul>
+                <li v-for="step in invalidSteps" :key="step" v-translate="stepTextKey(step)"></li>
+              </ul>
+              <template v-if="lastValidStep >= 1">
+                <span v-translate="'loadErrorStepsFromValidAction'" />
+                <span v-translate="stepTextKey(lastValidStep)" />.
+                <span v-translate="'loadErrorStepsFromValidWarning'" />
+              </template>
+              <span v-else v-translate="'loadErrorStepsAllAction'" />
+            </p>
+            <p v-else>{{ loadError }}</p>
```

Comment:
In the case where the state is invalid and there are invalid steps, the `loadError` is not very informative ("There was a problem loading your data. Some data may have been invalid. Please contact support if this issue persists.") so we replace it with the richer UI here.